### PR TITLE
Start release note for GWT 2.8

### PR DESCRIPTION
**Pull request by jDramaix**
work in progress

[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/137)

### CODE REVIEW

File: src/main/markdown/release-notes.md [hunk 1]
```diff
@@ -54,6 +55,55 @@ The GWT Release Notes
...
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_BETA"></a> Release Notes for 2.8.0 (Beta)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
+
+- GSS is not considered as experimental anynmore and old CssResource is now deprecated.
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: typo in _anymore_, also maybe put CssResource within backticks

File: src/main/markdown/release-notes.md [hunk 2]
```diff
@@ -54,6 +55,55 @@ The GWT Release Notes
...
+
+#### JDK emulation
+
+
+#### Logging and Stack Traces
+
+
+#### GSS support
+
+GSS is not experimental anymore but it's not enabled by default for easier and smoother migration to GWT 2.8. The `CssResource.enableGss` configuration property turns on GSS support. For more information on GSS migration, please refers to [this document]()
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: _please refer to_ (no _s_)

**Comment by branflake2267** [comment]
I like the sounds of this PR. :)

File: src/main/markdown/release-notes.md (hunk 1 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 2 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md [hunk 3]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
+- Long Division are 5x faster.
+- Double/Boolean are not boxed anymore.
```
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: We should maybe gather these items in one item ? Or be more generic

File: src/main/markdown/release-notes.md [hunk 4]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
 
 * * *
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Perhaps something like:
JsInterop is graduated from experimantal. See [final JsInterop specification]

File: src/main/markdown/release-notes.md [hunk 5]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+
+- GSS is no longer considered as experimental and old /CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated.
+
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Classic dev mode deprecated.
(and instructions for switching back test cases temporarily)

File: src/main/markdown/release-notes.md [hunk 6]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
+
+- GSS is no longer considered as experimental and old /CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated.
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Link to instructions for migrations

File: src/main/markdown/release-notes.md [hunk 7]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+
+### Compiler changes
+
+- Java 8 syntax is now supported by the compiler.
+- New optimizer framework that reruns the optimization process only on the part of the AST that were impacted by the changes since its last run. That speedup compilation of large application.
+-
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Added better type specialization that enables better optimizations.

File: src/main/markdown/release-notes.md [hunk 8]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+- Many bugfixes.
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Added -setProperty flag

File: src/main/markdown/release-notes.md [hunk 9]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
+- Long Division are 5x faster.
+- Double/Boolean are not boxed anymore.
+
+#### Logging and Stack Traces
+
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Improved logging and stack traces for superdev mode.

File: src/main/markdown/release-notes.md [hunk 10]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+- Many bugfixes.
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
+- Long Division are 5x faster.
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: New long emulation with signification performance improvements (e.g. divisions 50x faster)

File: src/main/markdown/release-notes.md [hunk 11]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+-
+- Many bugfixes.
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: Started using ES6 Maps when available for HashMap/HashSet that yields up 3x performance improvements.

File: src/main/markdown/release-notes.md [hunk 12]
```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
...
+
+#### Widgets
+
+#### Dev Mode
+
+#### Super Dev Mode
+ - SDM caches on disk the MinimalRebuildCache object, which speeds up the first SDM compile on a restart of the process.
+
+#### Testing
+
```
Comment (gkdn) [inline src/main/markdown/release-notes.md]: There will be some more items from with the latests submissions. Please don't forget to revisit.

File: src/main/markdown/release-notes.md (hunk 4 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 5 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 6 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 12 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 11 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 10 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 9 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md (hunk 8 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

File: src/main/markdown/release-notes.md [hunk 13]
```diff
@@ -54,6 +55,64 @@ The GWT Release Notes
...
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- JsInterop is graduated from eperimental. See [final JsInterop specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0).
+
+- GSS is no longer considered as experimental and old CssResource is now deprecated.
```
Comment (dankurka) [inline src/main/markdown/release-notes.md]: Add link to migration guide here

File: src/main/markdown/release-notes.md [hunk 14]
```diff
@@ -54,6 +55,64 @@ The GWT Release Notes
 **Note** - M1 = first milestone, RC1 = first release candidate
 
 * * *
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
```
Comment (dankurka) [inline src/main/markdown/release-notes.md]: Lets make this BETA1 for now

File: src/main/markdown/release-notes.md [hunk 15]
```diff
@@ -54,6 +55,64 @@ The GWT Release Notes
...
+- Java 8 syntax supported. 
+
+- JsInterop is graduated from eperimental. See [final JsInterop specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0).
+
+- GSS is no longer considered as experimental and old CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated. For more information on GSS migration, please refer to [this document](http://www.gwtproject.org/articles/gss_migration.html)
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
```
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: Not the case in beta1, so I remove it for the time being

File: src/main/markdown/release-notes.md (hunk 14 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: DONE

**Pull request merged**